src/docs/platforms/javascript/react/integrations/react-router.mdx

```diff
@@ -0,0 +1,170 @@
+    // Use custom history
+    <Router history={history}>
+      <App />
+    </Router>
+  );
+}
+```
+
+Now you should be generating pageload/navigation transactions from the `BrowserTracing` integration using react router instrumentation.
+
+### Parameterized transaction names
+
+To get parameterized transaction names (ex. `/teams/:team/user/:userid` instead of `/teams/123/user/345`), you must give the SDK access to the match path of the route render. This is as there is no static route config with that the SDK can use in React Router v4/v5. There are two ways to accomplish this:
+
+1. Pass a Route Config Object
+
+You can pass an array of route config objects as per [react-router-config](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-config).
+
+```js
+import {Route, Router, Switch, matchPath } from 'react-router-dom';
+import { createBrowserHistory } from 'history'; 
+import { reactRouterV5Instrumentation } from '@sentry/react';
+
+const history = createBrowserHistory();
+
+// Array of Route Config Objects
+const routes = [{ path: '/users/:userid' }, { path: '/users' }, { path: '/' }];
+
+Sentry.init({
+  tracesSampleRate: 1,
+  integrations: [
+    new Integrations.BrowserTracing({
+      routingInstrumentation: reactRouterV5Instrumentation(history, routes, matchPath),
+    });
+  ]
+});
+
+// In your App render:
+render() {
+  return (
+    <Router history={history}>
+      <Switch>
+         <Route path="/users/:userid" component={() => <div>UserId</div>} />
+         <Route path="/users" component={() => <div>Users</div>} />
+         <Route path="/" component={() => <div>Home</div>} />
+      </Switch>
+    </Router>
+  );
+}
+```
+
+2. Use Sentry Route component
+
+Use `withSentryRouting` higher order component to create a `SentryRoute` component that will accurately update the match path
+
+```
+import {Route, Router, Switch } from 'react-router-dom';
+import { createBrowserHistory } from 'history'; 
+import { reactRouterV4Instrumentation, withSentryRouting } from '@sentry/react';
+
+// Create Custom Sentry Route component
+const SentryRoute = withSentryRouting(Route);
+const history = createBrowserHistory();
+
+Sentry.init({
+  tracesSampleRate: 1,
+  integrations: [
+    new Integrations.BrowserTracing({
+      routingInstrumentation: reactRouterV4Instrumentation(history),
+    });
+  ]
+});
+
+render() {
+  return (
+    <Router history={history}>
+      <Switch>
+         <SentryRoute path="/users/:userid" component={() => <div>UserId</div>} />
+         <SentryRoute path="/users" component={() => <div>Users</div>} />
+         <SentryRoute path="/" component={() => <div>Home</div>} />
+      </Switch>
+    </Router>
+  );
+}
+```
+
+## React Router v3
+
+To use the router integration, import and set custom routing instrumentation and pass it the history, your routes and a match function. React Router v3 support is maintained for React Router >= 3.2.0 and < 4.0.0.
+
+See the typedoc for 
```

Comment:
Missing link + rest of sentence

Sorry, I know this is a draft, just want to help mark what's pending.